src/libraries/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -260,6 +260,10 @@ System.Reflection.PortableExecutable.ManagedPEBuilder</PackageDescription>
     <ProjectReference Include="$(LibrariesProjectRoot)System.Collections.Immutable\src\System.Collections.Immutable.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
+    <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
```

Comment:
`System.Memory` is already transitively referenced through the immutable collections package, I made the reference explicit.